.all-contributorsrc

```diff
@@ -14,9 +14,7 @@
 				"infra",
 				"maintenance",
 				"projectManagement",
-				"test",
-				"tool",
-				"bug"
+				"tool"
```

Comment:
🐛 These should be preserved...

Reply:
Found it: https://github.com/JoshuaKGoldberg/create-typescript-app/issues/2063 -> https://github.com/all-contributors/cli/issues/371